docs/releases/3.0.txt

```diff
@@ -320,6 +323,10 @@ Miscellaneous
   the session and :func:`django.contrib.auth.logout` no longer preserves the
   session's language after logout.
 
+* For compatibility with Python 3.7 new ``-k`` option for ``unittest``, the
```

Comment:
The `django-admin test -k` option now works as the `unittest -k` option rather than as a shortcut for `--keepdb`.